lib/optimizely/logger.rb

```diff
@@ -34,7 +34,8 @@ class SimpleLogger < BaseLogger
     # Simple wrapper around Logger.
 
     def initialize(min_level = Logger::INFO)
-      @logger = Logger.new(STDOUT)
+      super()
+      @logger = Logger.new($stdout)
```

Comment:
Is $stdout supposed to be defined somewhere above?

Reply:
This was one of the automated fixes generated by [rubocop](https://docs.rubocop.org/rubocop/cops_style.html#styleglobalstdstream). Looks like both `STDOUT` and `$stdout` have global scope, but `$stdout` is preferred because `STDOUT` is a constant.